bin/console

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env php
 <?php
 
-use Surfnet\StepupRa\Kernel;
+use src\Kernel;
```

Comment:
The other projects solved this by moving the kernel to the Surfnet\StepupRa folder. I much prefer this to introducing a bogus /src namespace. Can u fix?

Reply:
Thanks that looks much better!